publication-commons/src/main/java/no/unit/nva/publication/model/storage/FileDao.java

```diff
@@ -115,7 +116,7 @@ public URI getCustomerId() {
     @JsonProperty("identifier")
     @Override
     public SortableIdentifier getIdentifier() {
-        return identifier;
+        return getData().getIdentifier();
```

Comment:
This makes identifier be a part of row, found out today that it was not. 
<img width="719" alt="image" src="https://github.com/user-attachments/assets/7645517c-222d-4c96-91b3-5e049581fd9b" />

Reply:
Out of interest, is the field in use at all now?

Reply:
It is indirectly in use, cause in transactions we use primaryKey() method. 
```
    @Override
    @JsonProperty(PRIMARY_KEY_SORT_KEY_NAME)
    public String getPrimaryKeySortKey() {
        return KEY_PATTERN.formatted(TYPE, getIdentifier());
    }
```

It seems that that json annotations fixes it, but everything works fine when identifier on dao is null. Since all dao's have identifier field in database, I think it is great to have it here as well.